### PR TITLE
Added code for 'valueScopes' Authorization

### PR DESCRIPTION
**Pull request by WilcoLouwerse**

### CODE REVIEW

File: api/src/Service/EavService.php [hunk 1]
```diff
@@ -181,10 +181,6 @@ public function getObject(?string $id, string $method, Entity $entity, ?string $
                 ];
             }
 
-            if ($method == 'POST' || $method == 'PUT') {
-                return $this->objectEntityService->handleOwner($object, $owner);
-            }
-
```
Comment (rjzondervan) [inline api/src/Service/EavService.php:187]: Just checking: there is no way to do this somewhere earlier?

File: api/src/Service/AuthorizationService.php [hunk 2]
```diff
@@ -99,63 +240,307 @@ public function getScopesForAnonymous(): array
...
+        // Handle the result
+        if ($checkValueScopes['hasValueScopes']) {
+            if ($checkValueScopes['matchValueScopes']) {
+                return true;
+            } else {
+                $message = $checkValueScopes['error']['message'] ?? '';
+                $scope = $checkValueScopes['error']['failedScope'] ?? '';
+                $attributeName = $info['attribute'] ? $info['attribute']->getName() : ($checkValueScopes['error']['attribute'] ? $checkValueScopes['error']['attribute']->getName() : '');
+                $shouldMatchValues = '';
+                if ($checkValueScopes['error']['shouldMatchValues']) {
```
Comment (rjzondervan) [inline api/src/Service/AuthorizationService.php:266]: if if if...
Note to refactor this service ;)

**Review by rjzondervan — APPROVED** [review]
The authorizationservice needs a refactor, but okay for now

File: api/src/Service/AuthorizationService.php [hunk 3]
```diff
@@ -99,63 +240,307 @@ public function getScopesForAnonymous(): array
...
+                    $shouldMatchValues = count($checkValueScopes['error']['shouldMatchValues']) > 1 ? 'one of ' : '';
+                    $shouldMatchValues = $shouldMatchValues.'['.implode(', ', $checkValueScopes['error']['shouldMatchValues']).']';
+                }
```
Comment (rjzondervan) [inline api/src/Service/AuthorizationService.php:269]: if if if...
Note to refactor this service ;)

**Review by rjzondervan — APPROVED** [review]
Some parts need some refactoring, but okay for now

**Comment by WilcoLouwerse** [comment]
This merge has not super high prio and there are still some changes needed for this BL to be completed correctly.
See: https://conduction.atlassian.net/browse/TOP-61

**Pull request merged**